evalml/pipelines/utils.py

```diff
@@ -201,6 +202,21 @@ def _get_ohe(X, y, problem_type, estimator_class, sampler_name=None):
     return components
 
 
+def _get_ordinal_encoder(X, y, problem_type, estimator_class, sampler_name=None):
+    components = []
+
+    ordinal_cols = list(
+        X.ww.select(
+            ["Ordinal"],
+            return_schema=True,
+        ).columns,
+    )
+
+    if len(ordinal_cols) > 0:
+        components.append(OrdinalEncoder)
```

Comment:
Note: Unlike the OneHotEncoder, we do include the OrdinalEncoder when CatBoost is being used, because ordinal-encoding is not a CatBoost behavior, so we want to benefit from knowing the order of categories.